public/app.css

```diff
@@ -16,14 +16,22 @@ html, body {
 
 .verticalDivider {
     float: left;
+    box-sizing: border-box;
     border-right: 1px solid #000000;
     height: 100%;
-    min-width: 49px;
+    min-width: 1px;
 }
 
 .actionsSection {
     float: left;
-    width: calc(100% - 350px);
+    width: calc(100% - 300px - 1px);
```

Comment:
I understand what you are aiming for here, but I would prefer 299px instead of 300 - 1. Not a big problem though if you wish to go with your way.